dev-system/dev/user.clj

```diff
@@ -29,6 +32,26 @@
           (fn [app-map]
             (u/map-values #(assoc-in % [:log :level] level) app-map))))
 
+(defn set-mode!
+  "Set the mode to use when starting/resetting. Valid options are:
+
+  * :in-memory
+  * :external"
+  [new-mode]
+  (alter-var-root #'mode (constantly new-mode)))
+
+(defn use-legacy?
```

Comment:
I think question marks should be reserved for predicates, but I'm willing to bow to the crowd view here.